src/util.rs

```diff
@@ -48,6 +48,27 @@ pub fn history() -> web_sys::History {
 pub fn html_document() -> web_sys::HtmlDocument {
     wasm_bindgen::JsValue::from(document()).unchecked_into::<web_sys::HtmlDocument>()
 }
+/// Convenience function to access the `web_sys::HtmlCanvasElement`.
+pub fn canvas(id: &str) -> web_sys::HtmlCanvasElement {
+    let canvas = document().get_element_by_id(id).unwrap();
+    canvas
+        .dyn_into::<web_sys::HtmlCanvasElement>()
+        .map_err(|_| ())
+        .unwrap()
+}
+
+/// Convenience function to access the `web_sys::CanvasRenderingContext2d`.
+pub fn canvas_context(
+    canvas: &web_sys::HtmlCanvasElement,
+    context: &str,
+) -> web_sys::CanvasRenderingContext2d {
+    canvas
+        .get_context(context)
+        .unwrap()
+        .unwrap()
+        .dyn_into::<web_sys::CanvasRenderingContext2d>()
+        .unwrap()
+}
```

Comment:
I suggest to rename this function to `canvas_context_2d` and remove parameter `context`. Because user can send only `"2d"` as the `context` otherwise it fails during casting (https://developer.mozilla.org/en-US/docs/Web/API/HTMLCanvasElement/getContext).

I assume that the updated function shouldn't fail, but I recommend to at least replace `unwrap` with `expect` (if it make sense for you).

Reply:
Good point. Originally had it called that, but assumed this one was more general, without checking the casting.